components/src/core/drawer/drawer-body/nav-item/drawer-nav-item.component.scss

```diff
@@ -80,6 +80,22 @@ pxb-drawer-nav-item {
             transform: rotate(180deg);
         }
     }
+    .pxb-drawer-nav-item-rail {
+        display: flex;
+        flex-direction: column;
+        height: 72px;
+        width: 71px;
```

Comment:
72?

Reply:
Yeah this should be 72, ran into some issues with a 1px border causing some overflow.  I'll play around with this more.